### DISCUSSION TITLE
Possibilities for auth

### DISCUSSION THREAD

**Discussion by Getty** [langfuse/langfuse#2206] [Support] [answered]
What are the alternatives for the auth of the users? I hope no auth and SSO is planned :D

**Comment by marcklingen** [comment, marked as answer]
You can use the following options:

- email/password, self-contained, no external provider
- SSO via GitHub, Google, Auth0, AWS Cognito, Okta, ...

Auth is completely managed within the self-hosted deployment using Auth.js.

Currently there is no un-authed self-hosted deployment of Langfuse as most users use Langfuse collaboratively with multiple people signing into Langfuse.

What kind of change do you suggest?
